tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -150,8 +149,9 @@ def test_synchronization(folder_path, case, host):
         if (case == 'delete'):
             # Execute query to DB
             sleep(5)
-            result = query_db(host_manager, db_script, db_path, f'\"{query}\"')
-            assert not json.loads(result)
+            result = execute_wdb_query(query, 'wazuh-manager', host_manager)
+
+            assert result == '[]'
```

Comment:
Didn't this return `{'ok', []}`?

Reply:
I have checked it again to be sure, as you can see [here](https://github.com/wazuh/wazuh-qa/blob/0fa86091d2bbf35fdf53e3355d5234f5bd714f7e/tests/system/provisioning/scripts/wdb-query.py#L71) the code returns the result without the `ok `